specs/ceph_crud_flows.adoc

```diff
@@ -0,0 +1,557 @@
+          help: "Create rbd"
+          inputs:
+            mandatory:
+              - Rbd.pool_id
+              - Rbd.name
+              - Rbd.size
+          name: "Create Rbd"
+          run: tendrl.ceph_integration.objects.Rbd.atoms.create.Create
+          type: Create
+          uuid: 7a2df258-9b24-4fd3-a66f-ee346e2e3720
+        Delete:
+          enabled: true
+          help: "Delete rbd"
+          inputs:
+            mandatory:
+              - Rbd.pool_id
+              - Rbd.name
+          name: "Delete Rbd"
+          run: tendrl.ceph_integration.objects.Rbd.atoms.delete.Delete
+          type: Delete
+          uuid: 7a2df258-9b24-4fd3-a66f-ee346e2e3721
+        Resize:
+          enabled: true
+          help: "Resize Rbd"
+          inputs:
+            mandatory:
+              - Rbd.pool_id
+              - Rbd.name
+              - Rbd.size
+          name: "Resize Rbd"
+          run: tendrl.ceph_integration.objects.Rbd.atoms.resize.Resize
+          type: Update
+          uuid: 7a2df258-9b24-4fd3-a66f-ee346e2e3722
+      flows:
+        CreateRbd:
+          atoms:
+            - tendrl.ceph_integration.objects.Rbd.atoms.create.Create
+          help: "Create Rbd"
+          enabled: true
+          inputs:
+            mandatory:
+              - Rbd.pool_id
+              - Rbd.name
+              - Rbd.size
+          run: tendrl.ceph_integration.objects.Rbd.flows.create_rbd.CreateRbd
+          type: Create
+          uuid: 9bc41d8f-a0cf-420a-b2fe-18761e07f3d2
+        DeleteRbd:
+          atoms:
+            - tendrl.ceph_integration.objects.Rbd.atoms.delete.Delete
+          help: "Delete Rbd"
+          enabled: true
+          inputs:
+            mandatory:
+              - Rbd.pool_id
+              - Rbd.name
+          run: tendrl.ceph_integration.objects.Rbd.flows.delete_rbd.DeleteRbd
+          type: Delete
+          uuid: 4ac41d8f-a0cf-420a-b2fe-18761e07f3a7
+        ResizeRbd:
+          atoms:
+            - tendrl.ceph_integration.objects.Rbd.atoms.resize.Resize
+          help: "Resize Rbd"
+          enabled: true
+          inputs:
+            mandatory:
+              - Rbd.pool_id
+              - Rbd.name
+              - Rbd.size
+          run: tendrl.ceph_integration.objects.Rbd.flows.resize_rbd.ResizeRbd
+          type: Update
+          uuid: 4ac41d8f-a0cf-420a-b2fe-18761e07f3c9
+      attrs:
+        name:
+          help: Name of the rbd
+          type: String
+        size:
+          help: Size of the rbd (MB)
+          type: int
+        pool_id:
+          help: Id of the pool
+          type: String
+        order:
+          help: Order of the rbd
+          type: String
+        block_name_prefix:
+          help: Prefix for the block name
+          type: String
+        format:
+          help: format of rbd
+          type: int
+        features:
+          help: featutres for rbd
+          type: list
+        flags:
+          help: flags for rbd
+          type: list
```

Comment:
Of course there are but for the time being just supporting `size` only as exposed from tendrl.